scripts/ldap/replication/setupReplicationScript.sh

```diff
@@ -103,6 +119,16 @@ echo "olcMirrorMode: TRUE" >> scriptConfig.ldif
 ldapmodify -Y EXTERNAL  -H ldapi:/// -f scriptConfig.ldif
 rm scriptConfig.ldif
 
+op=$(ldapsearch -w $password -x -D cn=admin,cn=config -b olcDatabase={2}mdb,cn=config "olcSyncrepl")
+if [[ $op == *"olcSyncrepl:"* ]];then
+  ldapmodify -Y EXTERNAL  -H ldapi:/// -f cleanup/data.ldif
+fi
+
+op=$(ldapsearch -w $password -x -D cn=admin,cn=config -b olcDatabase={2}mdb,cn=config "olcMirrorMode")
+if [[ $op == *"olcMirrorMode:"* ]];then
+  ldapmodify -Y EXTERNAL  -H ldapi:/// -f cleanup/olcmirromode_data.ldif
```

Comment:
can you log success failure of these commands also print ldap commands at info level

Reply:
this is bash script added echo statements.info level ldap commands ?